internal/pkg/cli/app_deploy.go

```diff
@@ -49,15 +48,16 @@ type appDeployVars struct {
 type appDeployOpts struct {
 	appDeployVars
 
-	projectService     projectService
-	workspaceService   wsAppReader
-	ecrService         ecrService
-	dockerService      dockerService
-	s3Service          artifactPutter
-	runner             runner
-	appPackageCfClient projectResourcesGetter
-	appDeployCfClient  cloudformation.CloudFormation
-	sessProvider       sessionProvider
+	projectService   projectService
+	workspaceService wsAppReader
+	ecrService       ecrService
+	dockerService    dockerService
+	s3Service        artifactPutter
```

Comment:
just a tiny thing - can we call this artifactUploader or artifactStorer?

Reply:
I'll incorporate this in the next PR!